.gitignore

```diff
@@ -24,3 +24,10 @@
 
 # ignore local settings
 .idea/*
+
+# Ignore rubocop corrections
+.rubocop_todo.yml
```

Comment:
1. Why do you need this file?
1. Why is it ignored? Rubocop will fail because checked-out copies won't have this file